core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -135,126 +136,135 @@ object ConsoleProducer {
 
   class ProducerConfig(args: Array[String]) {
     val parser = new OptionParser(false)
-    val topicOpt = parser.accepts("topic", "REQUIRED: The topic id to produce messages to.")
+    val topicOpt = parser.accepts("topic", "The topic id to produce messages to.")
       .withRequiredArg
-      .describedAs("topic")
+      .describedAs("topic id")
```

Comment:
I think `topic` is better compared to `topic id`, since `topic id` is not often used in the community. Perhaps the above message can also be updated to remove `id`.